app/controllers/point_details_controller.rb

```diff
@@ -1,11 +1,13 @@
 class PointDetailsController < ApplicationController
-  before_action :require_resort_or_group_leader
   before_action :set_entities
-  before_action :changeable_evaluation
   before_action :validate_correct_group
 
   def update
-    point_detail_service = CreateOrUpdatePointDetail.new(@user, @principle, @evaluation)
+    unless EvaluationPolicy.new(current_user, @evaluation).submit_point_request?
+      raise Pundit::NotAuthorizedError, "not allowed to update? this #{@evaluation.inspect}"
+    end
```

Comment:
Here also I'd suggest this form.
```suggestion
    authorize @evaluation, :submit_point_request?
```